Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -71,14 +76,80 @@ HTMLAttachmentElement::~HTMLAttachmentElement() = default;
 
 Ref<HTMLAttachmentElement> HTMLAttachmentElement::create(const QualifiedName& tagName, Document& document)
 {
-    return adoptRef(*new HTMLAttachmentElement(tagName, document));
+    Ref attachment = adoptRef(*new HTMLAttachmentElement(tagName, document));
+    if (document.settings().attachmentWideLayoutEnabled() && DeprecatedGlobalSettings::attachmentElementEnabled()) {
```

Comment:
The check for `DeprecatedGlobalSettings::attachmentElementEnabled()` is unnecessary, since you already can't create `HTMLAttachmentElement`s if it's false.

(see `HTMLAttachmentElement.idl`)

Reply:
Oops, good point! I should rephrase the setting description then, from "wide-layout implies attachment element" to "... requires ...".